.github/workflows/release.yml

```
@@ -398,7 +398,7 @@ jobs:
         run: pnpm install --frozen-lockfile
 
       - name: Publish npm package as latest
-        run: npm publish packages/@biomejs/js-api --tag latest --access public --provenance
+        run: pnpm publish packages/@biomejs/js-api --no-git-checks --tag latest --access public
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

FYI, you can still enable provenance in pnpm by environment variable.

```suggestion
          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
          NPM_CONFIG_PROVENANCE: true
```